src/components/Box/Box.tsx

```diff
@@ -83,6 +83,7 @@ export const useStyles = makeStyles<BoxProps>(
       alignBaseline: { alignItems: 'baseline' },
       alignCenter: { alignItems: 'center' },
       alignStart: { alignItems: 'start' },
+      flexWrap: { flexWrap: ({ flexWrap }) => (flexWrap ? 'wrap' : 'nowrap') },
```

Comment:
Should this be:

```
flexWrap: { flexWrap: 'wrap' }
```

When I tried `flexWrap={false}` I was expecting `flex-wrap: nowrap` based on this logic to be applied to the element, but it appears it did not apply.  I could totally be missing something here too

![Screen Shot 2022-01-06 at 9 17 09 PM](https://user-images.githubusercontent.com/8069555/148480685-83a96b54-638e-4b69-894a-6991d6049c3b.png)

Reply:
Yeah, it looks like there's some double logic being applied. I could either switch this to your suggestion, or remove the clsx class to make this always apply. Will update.